module/motion/QuinticWalk/src/QuinticWalk.cpp

```diff
@@ -260,12 +274,14 @@ namespace module::motion {
                                                Eigen::Affine3f(left_foot.cast<float>()),
                                                Eigen::Affine3f(right_foot.cast<float>()));
 
-        auto waypoints = motionLegs(joints);
+        auto waypoints_legs = motion(joints);
+        emit(std::move(waypoints_legs));
 
-        emit(std::move(waypoints));
+        auto waypoints_arms = motion(arm_positions);
+        emit(std::move(waypoints_arms));
```

Comment:
You could combine these together into a single emit

Reply:
I have combined them.